### ISSUE TITLE
Staying in line with intranet policy : the test suite should not be public

### ISSUE THREAD

**Issue by geoffreylgv** [geoffreylgv/LearningHub#27]
Staying in line with intranet policy related to #13 the test suite should not be public

**Comment by geoffreylgv** [comment]
> Staying in line with intranet policy related to #13 the test suite should not be public

Hi @bravemaster3, on the main repo that refers to the testing material, there is no folder, according to the policy of the intranet

**Comment by bravemaster3** [comment]
Yes, I noticed that they have removed the simple shell folder of the test suite.
As soon as I have some time, I can readapt the sections where it was mentioned... Thanks.

Right now we don't break any ALX policies. We refered to the test suite (we didn't copy its content). 
But we definetely have to adjust the content so that useless links are taken down.

**Comment by geoffreylgv** [comment]
> Yes, I noticed that they have removed the simple shell folder of the test suite. As soon as I have some time, I can readapt the sections where it was mentioned... Thanks.
> 
> Right now we don't break any ALX policies. We refered to the test suite (we didn't copy its content). But we definetely have to adjust the content so that useless links are taken down.

Alright, thank you
